### PR TITLE
For PR integration tests, upload WSM logs to GitHub

### PR DESCRIPTION
**Pull request by melissachang**
This should make flaky tests easier to debug.

Example:

https://github.com/DataBiosphere/terra-workspace-manager/actions/runs/2005828712

### CODE REVIEW

**Review by zloery — APPROVED** [review]

**Review by jmczerk — APPROVED** [review]

**Comment by melissachang** [comment]
No downstream changes needed.

Upstream changed PR integration test action. This repo doesn't run integration tests on PRs. Only unit tests run for PRs. (Integration tests run nightly.)

File: .github/workflows/pr-integration.yml [hunk 1]
```diff
@@ -124,3 +124,12 @@ jobs:
         author_name: "integrationTest"
         icon_emoji: ":triangular_ruler:"
         fields: job, commit
+
+    - name: Archive WSM logs
```
Comment (jaycarlton) [inline .github/workflows/pr-integration.yml:128]: Maybe add a description that the logs are available at the bottom of the screen on the run page, and/or name this `Archive WSM Logs with Github Action -- See Below`

**Review by jaycarlton — COMMENTED** [review]
Very nice!

File: .github/workflows/pr-integration.yml (hunk 1 above)
Comment (jaycarlton) [reply .github/workflows/pr-integration.yml:128]: LGTM. I don't see Approve on the review widget; maybe they don't show it anymore when you already have enough approving reviewers.

**Pull request merged**